### PR TITLE
Fix bug of closing Kafka consumer multiple times + Fix travis-ci build

### PR DESCRIPTION
**Pull request by rezabaktash**
…closed only once.

### CODE REVIEW

**Comment by talebipour** [comment]
Please take a look at this issue:
https://github.com/travis-ci/travis-ci/issues/10290

**Comment by rezabaktash** [comment]
The build is fixed now. Please review this change. @talebipour

File: .travis.yml [hunk 1]
```diff
@@ -1 +1,7 @@
-language: java
\ No newline at end of file
+dist: xenial
+language: java
+jdk:
+  - openjdk8
+env:
+  - PATH=$(echo "$PATH" | sed -e 's/:\/usr\/local\/lib\/jvm\/openjdk11\/bin//')
```
Comment (talebipour) [inline .travis.yml]: Doesn't this work?
PATH="$JAVA_HOME/bin:$PATH"

**Review by talebipour — CHANGES_REQUESTED** [review]

File: .travis.yml (hunk 1 above)
Comment (rezabaktash) [reply .travis.yml]: Done. It worked.

**Pull request merged**